src/containers/SideNav/SideNav.js

```diff
@@ -98,144 +98,149 @@ function SideNav(props) {
       isFixedNav
     >
       <SideNavItems>
-        <SideNavMenu
-          defaultExpanded
-          renderIcon={TektonIcon}
-          title={intl.formatMessage({
-            id: 'dashboard.sideNav.tektonResources',
-            defaultMessage: 'Tekton resources'
-          })}
-        >
-          <SideNavMenuItem {...getMenuItemProps(getPath(urls.pipelines.all()))}>
-            Pipelines
-          </SideNavMenuItem>
-          <SideNavMenuItem
-            {...getMenuItemProps(getPath(urls.pipelineRuns.all()))}
-          >
-            PipelineRuns
-          </SideNavMenuItem>
-          <SideNavMenuItem
-            {...getMenuItemProps(getPath(urls.pipelineResources.all()))}
-          >
-            PipelineResources
-          </SideNavMenuItem>
-          <SideNavMenuItem {...getMenuItemProps(getPath(urls.tasks.all()))}>
-            Tasks
-          </SideNavMenuItem>
-          <SideNavMenuItem {...getMenuItemProps(urls.clusterTasks.all())}>
-            ClusterTasks
-          </SideNavMenuItem>
-          <SideNavMenuItem {...getMenuItemProps(getPath(urls.taskRuns.all()))}>
-            TaskRuns
-          </SideNavMenuItem>
-          <SideNavMenuItem
-            {...getMenuItemProps(getPath(urls.conditions.all()))}
+        <div>
```

Comment:
Unfortunately this produces invalid HTML as the `SideNavItems` component outputs a `ul`, and `SideNavMenu` / `SideNavMenuItem` are `li`s. We cannot have `ul > div > li`

I was able to achieve the desired result with CSS only:

```diff
diff --git a/src/containers/SideNav/SideNav.scss b/src/containers/SideNav/SideNav.scss
index 6b0fbc13..9c35c7f2 100644
--- a/src/containers/SideNav/SideNav.scss
+++ b/src/containers/SideNav/SideNav.scss
@@ -17,6 +17,19 @@ limitations under the License.
     border-color: $gray-80;
     color: $gray-10;

+    .bx--side-nav__items {
+      display: flex;
+      flex-direction: column;
+
+      .bx--side-nav__item.bx--side-nav__item--icon + .bx--side-nav__item:not(.bx--side-nav__item--icon) {
+        // this matches the first nav link after the menus, so either Import Resources or About depending on the isReadOnly flag
+        display: flex;
+        flex-direction: column;
+        flex-grow: 1;
+        justify-content: flex-end;
+      }
+
+      .bx--side-nav__item:last-child {
+        margin-bottom: 1rem;
+      }
+    }
+
     .bx--side-nav__icon {
       &:not(.bx--side-nav__submenu-chevron) {
```

This actually moves both 'Import Resources' and 'About' to the bottom of the nav which I think makes sense here.

Reply:
@AlanGreene 
Thanks for feedback :) 
I wonder if it is the intended behavior. (both 'Import Resources' and 'About' moves to the bottom)

Reply:
Yes that's intended, I mentioned it in my comment above.

> This actually moves both 'Import Resources' and 'About' to the bottom of the nav which I think makes sense here.

Reply:
@AlanGreene Cool! I thought your approach is better and reflect your feedback. Please check again.